specification/dimensions/usageunit.md

```diff
@@ -0,0 +1,106 @@
+# Usage Unit
+
+Usage Unit is the unit of measure for the usage or consumption of a resource or service. Usage Unit is used in conjunction with Quantity In Usage Unit to gain understanding of usage or consumption of a given product or service in an environment.
+
+The UsageUnit column MUST be present in the billing data. This column MUST be of type String and MUST NOT contain null values when the ChargeType is 'Usage'. UsageUnit should be expressed as a single unit of measure.  Units of measure used in UsageUnit SHOULD adhere to the values and format requirements specified in the Recommended Values section below.  The UsageUnit column MUST not be used as the basis for determining values related to any pricing or cost metric.
+
+## Column ID
+
+UsageUnit
+
+## Display name
+
+Usage Unit
+
+## Description
+
+The unit of measure for the usage or consumption of a resource or service
+
+The Usage Unit for a particular line item may differ from PricingUnit when providers use different units, unit increments, or columns to determine cost. The values in UsageUnit and QuantityInUsageUnit are often listed at a finer granularity or over a different time interval than the PricingUnit and QuantityInPricingUnit. While UsageUnit may appear to relate to pricing and cost, UsageUnit is focused on resource or service consumption.  It is essential not to confuse UsageUnit with PricingUnit which is the basis for determining cost.
```

Comment:
Style wise, we should stick to calling it `Usage Unit` or `UsageUnit` consistently . It makes sense to refer to the human readable name in the intro, but going around the horn, it feels like we refer to the column name after that.


```suggestion
The UsageUnit for a particular line item may differ from PricingUnit when providers use different units, unit increments, or columns to determine cost. The values in UsageUnit and QuantityInUsageUnit are often listed at a finer granularity or over a different time interval than the PricingUnit and QuantityInPricingUnit. While UsageUnit may appear to relate to pricing and cost, UsageUnit is focused on resource or service consumption.  It is essential not to confuse UsageUnit with PricingUnit, which is the basis for determining cost.
```

Reply:
Please use **PricingMeasure** (non-normalized) instead of PricingUnit (semi-normalized)